src/counter/src/azurefiles/counter/App Resources/volume.yaml

```diff
@@ -1,5 +1,5 @@
 volume:
-  schemaVersion: 1.0.0-preview1
+  schemaVersion: 1.0.0-preview2
```

Comment:
This indentation is incorrect. Both provider and azureFileParameters shou;ld be under properties.

Reply:
Looks it is GitHub bug that by clicking expand, the formatting is wrong on expanding part. View file would show normal indentation.